Gemfile

```diff
@@ -2,7 +2,12 @@
 
 source 'https://rubygems.org'
 
-gem 'temporalio'
+# Primary Temporal dependency
+gem 'temporalio', path: '../temporal-sdk-ruby/temporalio'
+
+# Additional dependencies for certain samples
+gem 'google-protobuf', '~> 3.25'
```

Comment:
Fix this invalid path-specific dependency when https://github.com/temporalio/sdk-ruby/pull/213 released

Reply:
Fixed